Jenkinsfile

```diff
@@ -278,11 +278,7 @@ try {
 						if (hasRegressionRunSucceeded) {
 							println 'Regression suite passed, proceeding to next stage...'
 						} else {
-							// TODO: Regression suite is currently inconsistent, so failing builds automatically
-							// would cause false negatives. Uncomment the line below when the regression suite is
-							// more consistent
-
-							// error('Regression suite failed, check the CloudWatch logs above for more details')
+							error('Regression suite failed, check the CloudWatch logs above for more details')
```

Comment:
Curious on your thoughts, not necessarily looking for any changes here or elsewhere.

I wonder if we want to initially (or always) yield to an operator when the regression suite _fails_? Instead, we might alert via slack message and enter a waiting state for the operator to make a decision as to whether the deployment should proceed. It's not difficult to imagine scenarios where we're deploying changes that are more time-sensitive, and being forced to re-deploy due to an unrelated regression suite failure would be pretty frustrating.

Reply:
This is something I've been thinking about as well, and probably needs additional work. I was thinking about something more automated -- maybe if a run fails against the baseline, we run again to be _very_ sure that there wasn't some one-off problem that caused an erroneous failure? Then, if that "safetynet" run fails, we actually fail the deployment.

I also really like the idea of having an operator come in and make a decision to continue even if we fail as a sort of override and safeguard against potential false negatives.

I think both strategies are mutually exclusive from each other, and could be used together. Either way, I also feel a little uneasy now about straight-up failing. I think we should create a card for improving this.